zing_stats.py

```diff
@@ -970,23 +1236,22 @@ def configure_logging(args):
     logging.getLogger().setLevel(logging.DEBUG)
 
     ch = logging.StreamHandler(stream=sys.stderr)
+    ch.setLevel(logging.INFO)
     if args.log_quietly:
         ch.setLevel(logging.ERROR)
     elif args.log_verbosely:
         ch.setLevel(logging.DEBUG)
-    else:
-        ch.setLevel(logging.INFO)
 
     ch_format = logging.Formatter('%(message)s')
     ch.setFormatter(ch_format)
     logging.getLogger().addHandler(ch)
 
     if not args.no_logfile:
         fh = logging.FileHandler(args.logfile, delay=True)
-        if args.log_verbosely:
-            fh.setLevel(logging.DEBUG)
-        else:
-            fh.setLevel(logging.INFO)
+        fh.setLevel(logging.DEBUG)
+        # TODO, add custom trace level for json dumps
+        # if args.log_trace:
+        #     fh.setLevel(TRACE_LOGGING_LEVEL)
```

Comment:
Did you intend on debug logging to be the default if using a log file?

Reply:
Yes, in practice its needed for dev/test at least
Maybe need to add a trace option aswell but I tend to have to turn this on each time so leaving for now